doc/2d_combine.rst

```diff
@@ -0,0 +1,103 @@
+value of ``comb_id`` will be combined.
+
+Calibrations
+============
+
+Each calibration frame in the :ref:`pypeit_file:Data Block` should have the same ``calib`` ID value of
+the science data that uses it, or be set to ``all`` if used by all of the science frames
+in the Pypeit file.
+
+For the calibration frames ``comb_id`` is irrelevant and its value should be set to ``-1``.
+
+Example
+=======
+Here is an example of a Pypeit file where the science frames are combined ::
+
+    |                  filename |                 frametype |                 ra |                dec |  target | dispname | decker | binning |          mjd |    airmass | exptime |     dispangle |      amp | filter1 |    dateobs |         utc | calib | comb_id | bkg_id |
+    | DE.20170425.09554.fits.gz |                  arc,tilt |  57.99999999999999 |               45.0 | unknown |    1200G |  dra11 |     1,1 | 57868.110529 | 1.41291034 |     1.0 | 7699.95654297 | SINGLE:B |   OG550 | 2017-04-25 | 02:39:14.41 |   all |      -1 |     -1 |
+    | DE.20170425.09632.fits.gz | pixelflat,illumflat,trace |  57.99999999999999 |               45.0 | unknown |    1200G |  dra11 |     1,1 | 57868.111418 | 1.41291034 |    12.0 | 7699.95654297 | SINGLE:B |   OG550 | 2017-04-25 | 02:40:32.06 |   all |      -1 |     -1 |
+    | DE.20170425.09722.fits.gz | pixelflat,illumflat,trace |  57.99999999999999 |               45.0 | unknown |    1200G |  dra11 |     1,1 | 57868.112443 | 1.41291034 |    12.0 | 7699.95654297 | SINGLE:B |   OG550 | 2017-04-25 | 02:42:02.26 |   all |      -1 |     -1 |
+    | DE.20170425.09803.fits.gz | pixelflat,illumflat,trace |  57.99999999999999 |               45.0 | unknown |    1200G |  dra11 |     1,1 | 57868.113392 | 1.41291034 |    12.0 | 7699.95654297 | SINGLE:B |   OG550 | 2017-04-25 | 02:43:23.16 |   all |      -1 |     -1 |
+    | DE.20170425.50487.fits.gz |                   science | 260.04999999999995 | 57.958444444444446 |   dra11 |    1200G |  dra11 |     1,1 | 57868.584271 |  1.2765523 |  1200.0 | 7699.95654297 | SINGLE:B |   OG550 | 2017-04-25 | 14:01:27.15 |     0 |       1 |     -1 |
+    | DE.20170425.51771.fits.gz |                   science | 260.04999999999995 | 57.958444444444446 |   dra11 |    1200G |  dra11 |     1,1 | 57868.599136 | 1.29137753 |  1200.0 | 7699.95654297 | SINGLE:B |   OG550 | 2017-04-25 | 14:22:51.01 |     0 |       1 |     -1 |
+    | DE.20170425.53065.fits.gz |                   science | 260.04999999999995 | 57.958444444444446 |   dra11 |    1200G |  dra11 |     1,1 |   57868.6141 | 1.31412428 |  1000.0 | 7699.95654297 | SINGLE:B |   OG550 | 2017-04-25 | 14:44:25.52 |     0 |       1 |     -1 |
+
+The three science frames are combined together, therefore they are assigned a common value of ``comb_id``.
+Also the ``calib`` value is assigned to be the same for all the science frames, which will
+be reduced using the same calibrations (``calib`` = ``all`` for calibration frames).
+
```

Comment:
The only slightly confusing thing is the calib = 0 for the science frames. For a situation where we use i.e. separate tilts or arcs from the science frames this makes sense, but here the science frames are not used for any calibration so the 0 is a bit confusing. This is a flaw in our model, not the docs, but maybe you could add a sentence here clarifying that the zero does not do anything, but it would do something if tilt were set to the science frames (i.e. as it would be if we were using OH sky lines to determine tilts).

Reply:
Done!